identity/src/commonMain/kotlin/com/android/identity/util/ByteStringUtil.kt

```diff
@@ -0,0 +1,1670 @@
+ *
+ * @throws IllegalArgumentException if the [value] is outside the [validRange].
+ *
+ * @sample appendInt64LeSample
+ */
+fun ByteStringBuilder.appendInt64Le(
+    value: Long,
+    validRange: LongRange = Long.MIN_VALUE..Long.MAX_VALUE
+): ByteStringBuilder {
+    validateLong(value, validRange)
+    append((value shr 0).toByte())
+    append((value shr 8).toByte())
+    append((value shr 16).toByte())
+    append((value shr 24).toByte())
+    append((value shr 32).toByte())
+    append((value shr 40).toByte())
+    append((value shr 48).toByte())
+    append((value shr 56).toByte())
+    return this
+}
+
+/**
+ * Appends an 8-bit unsigned integer to this [ByteStringBuilder].
+ *
+ * This function appends the [value] as a single byte to the end of this [ByteStringBuilder].
+ * It also validates that the provided [value] is within the specified [validRange].
+ * By default the valid range is from [UByte.MIN_VALUE] to [UByte.MAX_VALUE].
+ *
+ * @param value The unsigned byte value to append.
+ * @param validRange The valid range for the unsigned byte value. Defaults to [UByte.MIN_VALUE]..[UByte.MAX_VALUE].
+ * @return This [ByteStringBuilder] instance, allowing for chaining of operations.
+ *
+ * @throws IllegalArgumentException if the [value] is outside the [validRange].
+ *
+ * @sample appendUInt8UByteSample
+ */
+fun ByteStringBuilder.appendUInt8(
+    value: UByte,
+    validRange: UIntRange = UByte.MIN_VALUE..UByte.MAX_VALUE
+): ByteStringBuilder {
+    validateUByte(value, validRange)
+    append(value.toByte())
+    return this
+}
+
+/**
+ * Appends an 8-bit unsigned integer to this [ByteStringBuilder].
+ *
+ * This function appends the [value] as a single byte to the end of this [ByteStringBuilder].
+ * It also validates that the provided [value] is within the specified [validRange].
+ * By default, the valid range is from [UByte.MIN_VALUE] to [UByte.MAX_VALUE].
+ *
+ * @param value The unsigned integer value to append.
+ * @param validRange The valid range for the unsigned integer value. Defaults to [UByte.MIN_VALUE]..[UByte.MAX_VALUE].
+ * @return This [ByteStringBuilder] instance, allowing for chaining of operations.
+ *
+ * @throws IllegalArgumentException if the [value] is outside the [validRange].
+ *
+ * @sample appendUInt8UIntSample
+ */
+fun ByteStringBuilder.appendUInt8(
+    value: UInt,
+    validRange: UIntRange = UByte.MIN_VALUE..UByte.MAX_VALUE
+): ByteStringBuilder {
+    validateUInt(value, validRange)
+    append(value.toByte())
+    return this
+}
+
+/**
+ * Appends an 8-bit unsigned integer to this [ByteStringBuilder].
+ *
+ * This function appends the [value] as a single byte to the end of this [ByteStringBuilder].
+ * It also validates that the provided [value] is within the specified [validRange].
+ * By default, the valid range is from [UByte.MIN_VALUE] to [UByte.MAX_VALUE].
+ *
+ * @param value The unsigned integer value to append.
```

Comment:
"unsigned integer" -> "integer"

...Maybe also on the next line, too.

And same for the other Int variations of the similar functions below (appendUInt16, appendUInt16Le, appendUInt32, etc).